### DISCUSSION TITLE
AttributeError: partially initialized module 'flet' has no attribute 'Page'

### DISCUSSION THREAD

**Discussion by isah-usman** [flet-dev/flet#815] [Q&A] [answered]
Hi Everyone,

I just stumbled upon Flet project and can't wait to get going with it.

I installed Python 3.10.0 and Flet 0.3.2 (see screenshot)

But I still could'nt run the first "Hello World" demo code I found in the flet.dev documentation.

The Error in the terminal:
AttributeError: partially initialized module 'flet' has no attribute 'Page' (see screenshot)

I also run:
`pip install flet --upgrade`

But still same issue.

If anyone has an idea, I'd greatly appreciate it!

Thank you so much.
[AttributeError Screenshot](https://github.com/isah-usman/myrepo/blob/main/Flet_Error.PNG)

**Comment by FeodorFitsner** [comment]
Could you run `pip show flet` and paste results here?

**Reply by isah-usman** [reply]
This is the result:
C:\Users\CTO003>pip show flet
Name: flet
Version: 0.3.2
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page:
Author:
Author-email: Appveyor Systems Inc. <hello@flet.dev>
License: MIT
Location: C:\Users\CTO003\AppData\Local\Programs\Python\Python311\Lib\site-packages
Requires: beartype, oauthlib, packaging, repath, requests, watchdog, websocket-client
Required-by:

**Comment by FeodorFitsner** [comment]
OK, what about these commands then?

```
where pip
where python
where flet
```

**Reply by isah-usman** [reply]
Okay, here are the results:

C:\Users\CTO003>where pip
C:\Users\CTO003\AppData\Local\Programs\Python\Python311\Scripts\pip.exe
C:\Users\CTO003\AppData\Local\Programs\Python\Python310\Scripts\pip.exe

C:\Users\CTO003>where python
C:\Users\CTO003\AppData\Local\Programs\Python\Python311\python.exe
C:\Users\CTO003\AppData\Local\Programs\Python\Python310\python.exe
C:\Users\CTO003\AppData\Local\Microsoft\WindowsApps\python.exe

C:\Users\CTO003>where flet
C:\Users\CTO003\AppData\Local\Programs\Python\Python311\Scripts\flet.exe
C:\Users\CTO003\AppData\Local\Programs\Python\Python310\Scripts\flet.exe

**Comment by FeodorFitsner** [comment, marked as answer]
Alright, you have a few Python installations. Try cleaning everything up like removing all pythons and folder leftovers and install only a single Python distro (3.11 is OK)?

Then install `flet` package again. Both `pip list` and `pip show flet` should yield results.

**Reply by isah-usman** [reply]
Everything works as expected 💯 also I have to change my project file name from "flet.py" to "hello.py"
**Thanks a lot Mr. Feodor**

**Reply by FeodorFitsner** [reply]
Oh, right! That should have been my first guess - your program name should not be `flet.py`! 🤗
